osrf/vrx
# Buoy update

This PR continues the work done in #699 with the rest of the round buoys. It also fixes the orientation of the Sydney Regatta buoys.

### How to test it?

Launch any of the worlds and verify that the buoys are still OK. E.g.:

```
ros2 launch vrx_gz competition.launch.py world:=sydney_regatta
```
